content/en/docs/grpc/_index.md

```diff
@@ -12,8 +12,4 @@ The current APIs are:
 - [schema definition](outputs): get or subscribe to Falco output events.
 - [schema definition](version): retrieve the Falco version.
 
-In order to interact with these APIs, the falcosecurity organization provides some clients/SDKs:
-
-- [client-go](./client-go)
-- [client-py](./client-py)
-- [client-rs](https://github.com/falcosecurity/client-rs)
+In order to interact with these APIs, the The Falco Project provides a [Golang SDK](./client-go).
```

Comment:
There was a discussion on LinkedIn where the user was using `client-go`. Has it been deprecated now?